scripts/bootstrap.sh

```diff
@@ -20,7 +20,7 @@ install_mangohud() {
 
 install_flatpaks () {
 	echo "INFO: installing Flatpaks, this will take a while..."
-	pacman -S flatpak 
```

Comment:
I want the `install_flatpak` function to be self-contained and not assume that Flatpak is already installed and configured. I guess it could have a $(which flatpak) and error if its not there, but the `pacman -S` was mostly just a placeholder until we supported more distros

Reply:
I added a check to see if `$(which flatpak)` returns anything